### PR TITLE
AndroidX Support

### PR DESCRIPTION
**Pull request by AppleEducate**

### CODE REVIEW

**Comment by jonasbark** [comment]
huh, again?

**Comment by AppleEducate** [comment]
Yeah sorry the other was for build grade, and this was for android X. I didn’t realize it needed both for the latest version of gradle.

**Comment by jonasbark** [comment]
can you merge the latest master branch from this repo to yours? e.g. I added some entries to the gitignore

File: android/build.gradle [hunk 1]
```diff
@@ -65,4 +66,4 @@ dependencies {
     api 'com.stripe:stripe-android:6.1.2'
 }
 
-apply plugin: 'kotlin-android-extensions'
\ No newline at end of file
+apply plugin: 'kotlin-android-extensions'
```
Comment (jonasbark) [inline android/build.gradle]: I think that can be removed altogether

Comment (AppleEducate) [reply android/build.gradle]: Yep, i removed it and it still built

**Comment by AppleEducate** [comment]
What is weird about the Android version is that it builds the amok successfully but crashes when run

**Comment by AppleEducate** [comment]
I looked at the code and the example for android is not using the flutter view

**Pull request merged**